api/main_endpoints/routes/User.js

```diff
@@ -199,6 +207,73 @@ router.post('/getPagesPrintedCount', (req, res) => {
   });
 });
 
+// Verifies the users session if they have an active jwtToken.
+// Used on the inital load of root '/'
+// Returns the name and accesslevel of the user w/ the given access token
+router.post('/verify', function(req, res) {
+  if (!checkIfTokenSent(req)) {
+    return res.sendStatus(UNAUTHORIZED);
+  }
+
+  const decoded = checkIfTokenValid(req);
+  if (decoded) {
+    res.status(OK).send(decoded);
+  } else {
+    res.sendStatus(UNAUTHORIZED);
+  }
+});
+
+router.get('/callback', async function(req, res) {
+  const code = req.query.code;
+  const email = req.query.state;
+  axios
+    .post('https://discord.com/api/oauth2/token',
```

Comment:
perhaps make a helper function in `main_endpoints/util` to genreate this link

Reply:
isn't link auto-generated by discord developer app I created? not sure how I'd implement this

Reply:
sorry this comment is for `api/main_endpoints/routes/User.js` line 231, not sure why it's here

Reply:
still not really understanding, I get you're talking about this part...
``client_id=${discordApiKeys.CLIENT_ID}` +
      `&client_secret=${discordApiKeys.CLIENT_SECRET}` +
      `&grant_type=client_credentials&code=${code}` +
      `&redirect_uri=${discordApiKeys.REDIRECT_URI}``
I don't get how you'd make a helper function because again, Discord is automatically generating this link